<launch>
    <arg name="pan_goal_position" default="75"/>
    <arg name="tilt_goal_position" default="180"/>
    <arg name="pan_goal_velocity" default="50"/>
    <arg name="tilt_goal_velocity" default="50"/>

    <rosparam file="$(find steve_pan_tilt_controller)/config/dynamixel_motors.yaml" command="load"/>

    
    <node name="steve_pan_tilt_controller" pkg="steve_pan_tilt_controller" type="steve_pan_tilt_controller.py" output="screen">
        <param name="config_file" value="$(find steve_pan_tilt_controller)/config/dynamixel_motors.yaml"/>
        <param name="pan_goal_position" value="$(arg pan_goal_position)"/>
        <param name="tilt_goal_position" value="$(arg tilt_goal_position)"/> 
        <!-- <param name="pan_goal_velocity" value="$(arg pan_goal_velocity)"/> 
        <param name="tilt_goal_velocity" value="$(arg tilt_goal_velocity)"/>  -->
    </node>
</launch>
